src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -160,6 +160,22 @@ def mgmt_if_entry_table_state_db(if_name):
 
     return b'MGMT_PORT_TABLE|' + if_name
 
+def get_sai_id_key(namespace, sai_id):
+    sai_id = sai_id.decode()
+    if namespace != '':
+        key = namespace + ':' + sai_id
+    else:
+        key = sai_id
+    return key.encode()
+
+def split_sai_id_key(sai_id_key):
+    sai_id_key = sai_id_key.decode()
+    if ':' in sai_id_key:
+        namespace, sai_id = sai_id_key.split(':')
```

Comment:
>split(':') [](start = 39, length = 10)

Why not just `split(b':')` without decoding?

Reply:
Modified as per comment.